### DISCUSSION TITLE
access session storage

### DISCUSSION THREAD

**Discussion by RezaOptic** [seleniumbase/SeleniumBase#2974] [General]
Hi,
is there any way, I can access session data after I login into a website? 
I tried using save_cookies but it doesn't contain the keys & values that exist in the browser session storage.
![Screenshot 2024-07-29 at 11 30 14 PM](https://github.com/user-attachments/assets/6e2d9f16-d891-479e-b855-3fa1c08c40a5)

**Comment by mdmintz** [comment]
Here are the Session Storage and Local Storage methods:

```python
self.set_session_storage_item(key, value)
self.get_session_storage_item(key)
self.remove_session_storage_item(key)
self.clear_session_storage()
self.get_session_storage_keys()
self.get_session_storage_items()

self.set_local_storage_item(key, value)
self.get_local_storage_item(key)
self.remove_local_storage_item(key)
self.clear_local_storage()
self.get_local_storage_keys()
self.get_local_storage_items()
```
